### PR TITLE
Added Ms Sql Server timestamp support.

### PR DESCRIPTION
**Pull request by AntonD-W**

### CODE REVIEW

**Comment by JonathanMagnan** [comment]
Thank you for your pull,

One of my developers will review it this week.

Best Regards,

Jon

---

**Performance Libraries**
`context.BulkInsert(list, options => options.BatchSize = 1000);  `
[Entity Framework Extensions](http://entityframework-extensions.net/) • [Entity Framework Classic](http://entityframework-classic.net/) • [Bulk Operations](http://bulk-operations.net/) •  [Dapper Plus](http://dapper-plus.net/)

**Runtime Evaluation**
`Eval.Execute("x + y", new {x = 1, y = 2}); // return 3`
[C# Eval Function](http://eval-expression.net/) • [SQL Eval Function](http://eval-sql.net/)

File: Main/Source/NMemory.Shared/NMemoryManage.cs [hunk 1]
```diff
@@ -5,5 +5,7 @@ namespace NMemory
     public static class NMemoryManager
     {
 		public static bool UseDefaultForNotNullable { get; set; }
+
+        public static bool UseSqlServerTimestamp { get; set; }
```
Comment (AntonD-W) [inline Main/Source/NMemory.Shared/NMemoryManage.cs:9]: Do you think this should be done as a separate setting, or maybe just add a reverse? I don't know any databases other than SqlServer that use this format.
If there are no such databases, then the point of setting is only to support backward compatibility. But even in this case, there will be support for the wrong format.

**Comment by JonathanMagnan** [comment]
Hello @AntonD-W ,

The method `GetLastTimestamp` makes sense.

However, indeed, as you commented, I believe added a static bool such as `ReverseByte` would be a better idea than specifying a provider.

I will merge your pull and make the change. It should be released later tonight or tomorrow.

Best Regards,

Jon

**Comment by AntonD-W** [comment]
Thank you @JonathanMagnan

**Comment by JonathanMagnan** [comment]
Hello @AntonD-W ,

The v3.1.2 has been released.

Honestly, I didn't know how to name this options so well, don't judge the name: `UseTimestampBytesReverse`. If you have a better one, I believe it's time to change it since you are the only one that will use it for now.

Let me know if everything works as you wish.

Best Regards,

Jon

**Comment by AntonD-W** [comment]
Hello @JonathanMagnan 

Normal name ```UseTimestampBytesReverse```. We have updated the packages, everything is working as it should. Thank.

**Pull request merged**